smartclient-core/src/main/java/org/srg/smartclient/JDBCHandler.java

```diff
@@ -136,6 +136,78 @@ protected DSResponse handleFetch(DSRequest request) throws Exception {
         return r[0];
     }
 
+    @Override
+    protected DSResponse handleAdd(DSRequest request) throws Exception {
+        if (! (request.getData() instanceof Map)) {
+            throw new RuntimeException("Bad request: operation 'ADD', the map of modified and PK fields " +
+                    "must be provided in the  'data' field.");
+        }
+
+        final OperationBinding operationBinding = getEffectiveOperationBinding(DSRequest.OperationType.FETCH, request.getOperationId());
+        final SQLAddContext<JDBCHandler> sqlAddContext = new SQLAddContext<>(this, request, operationBinding);
+
+
+        // --
+        final DSResponse[] response = {null};
+
+        policy.withConnectionDo(this.getDataSource().getDbName(), conn-> {
+
+            try (PreparedStatement st = conn.prepareStatement(sqlAddContext.getAddSQL(), ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY)) {
```

Comment:
This statement is not supposed to use anyResultSet, are you sure that we need  ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY ?